### PR TITLE
Add check boxes for individual expansions in debug-ui.

### PR DESCRIPTION
**Pull request by kberg**
image
image

### CODE REVIEW

File: compile-vue-templates.js [hunk 1]
```diff
@@ -3,348 +3,347 @@
  * Provide a mock vue so we can interact with
  * the components from node
  */
-const fs = require("fs");
-const beautify = require("js-beautify").js;
-const compiler = require("vue-template-compiler");
-const vue = require.resolve("vue");
-const dialogPolyfill = require.resolve("dialog-polyfill");
+const fs = require('fs');
```
Comment (kberg) [inline compile-vue-templates.js:6]: This reformat was supposed to be caught in a prior PR. Let me investigate.

**Comment by nwai90** [comment]
At 75% zoom, there is an extra `<` character after Ares:
Screenshot 2020-11-20 at 10 41 06 AM

At normal 100% zoom, text spills over and icons are cut off:
Screenshot 2020-11-20 at 10 40 24 AM

**Comment by kberg** [comment]
> At 75% zoom, there is an extra `<` character after Ares:
> Screenshot 2020-11-20 at 10 41 06 AM
> 
> At normal 100% zoom, text spills over and icons are cut off:
> Screenshot 2020-11-20 at 10 40 24 AM

Fixed the <.

**Comment by kberg** [comment]
> > At 75% zoom, there is an extra `<` character after Ares:
> > Screenshot 2020-11-20 at 10 41 06 AM
> > At normal 100% zoom, text spills over and icons are cut off:
> > Screenshot 2020-11-20 at 10 40 24 AM
> 
> Fixed the <.

Yeah I have similar problems, just at higher zoom rates. For now I'm not going to fix that, since this is just a debug page. I grossly stole from the create-game-form page.

**Review by alrusdi — APPROVED** [review]

**Comment by kberg** [comment]
Ugh, I merged this into the wrong branch.

**Pull request merged**